content/en/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md

```diff
@@ -299,7 +299,7 @@ The SNI proxy will forward the traffic to port `443`.
 1.  The following command will generate `istio-egressgateway-with-sni-proxy.yaml` which you can optionally edit and then deploy.
 
     {{< text bash >}}
-    $ cat <<EOF | helm template install/kubernetes/helm/istio/ --name istio-egressgateway-with-sni-proxy --namespace istio-system -x charts/gateways/templates/deployment.yaml -x charts/gateways/templates/service.yaml -x charts/gateways/templates/serviceaccount.yaml -x charts/gateways/templates/autoscale.yaml -x charts/gateways/templates/role.yaml -x charts/gateways/templates/rolebindings.yaml --set global.istioNamespace=istio-system -f - > ./istio-egressgateway-with-sni-proxy.yaml
+    $ cat <<EOF | istioctl manifest generate --set values.global.istioNamespace=istio-system -f - > ./istio-egressgateway-with-sni-proxy.yaml
     gateways:
```

Comment:
not sure if generate will take input like this.

Reply:
I don't think so. The '-' file is something that isn't Go internal afaik, we have to make it mean "stdin".

Reply:
@ostromart can you change this to something correct in a followup PR?

Reply:
TODO